.github/workflows/msys2_build.yml

```diff
@@ -16,10 +14,13 @@ jobs:
   build:
     runs-on: windows-latest
     strategy:
+      fail-fast: false
       matrix:
         sys:
           - mingw64
           - ucrt64
+        imtl: [main, v23.04]
```

Comment:
This is on the main branch, not understand why v23.04 is need here.

Reply:
Hi, 

v23.04 is tied to release tag (e.g https://github.com/OpenVisualCloud/Media-Transport-Library/releases/tag/v23.04)

we can define the branch/tag we need, but intention here is to tests against mainline and latest stable IMTL release which is currently v23.04

Reply:
The test for v23.04 should be moved to v23.04 branch, not here.